mmr/util.go

```diff
@@ -21,63 +17,3 @@ func (m MemStore) append(pos uint64, elem [][]byte) {
 func (m MemStore) getElem(pos uint64) []byte {
 	return m[pos]
 }
-
```

Comment:
@oshorefueled I have removed every unused type and functions from the code, check if anything wrong

Reply:
looks good to me